### DISCUSSION TITLE
Welcome to SIDES-Coding-Club Discussions!

### DISCUSSION THREAD

**Discussion by renixyyj** [renixyyj/SIDES-Coding-Club#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by devansSIDES** [comment]
Here are some Links for 
Coding Club **Zoom** Link - https://sides.zoom.us/j/86977615176

Software to Install 
- https://git-scm.com/downloads
- https://code.visualstudio.com
	
Github Group	
- https://github.com/devansSIDES/SIDES-Coding-Club
